### PR TITLE
[mqtt] Fix build errors in mqtt pipeline

### PR DESCRIPTION
**Pull request by dmolokanov**
Make sure MQTT pipeline succeeds for both linux and windows amd64 architectures

### CODE REVIEW

File: mqtt/build/windows/openssl.ps1 [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft. All rights reserved.
```
Comment (arsing) [inline mqtt/build/windows/openssl.ps1]: Why does it need openssl?

Comment (dmolokanov) [reply mqtt/build/windows/openssl.ps1]: some script uses it. I just copied all related scripts to build a folder. don't we need it for the same reason as edgelet? or for edgelet it is just to build with libiothsm?

Comment (arsing) [reply mqtt/build/windows/openssl.ps1]: iotedged needs openssl. mqtt code does not.

Yes, the existing Rust install script also installs openssl, but that doesn't mean we want to install openssl for the mqtt code. It's just going to slow down the build for no reason. You should instead edit the script to make installing openssl optional.

In fact I see you just wholesale copied all the scripts for edgelet to this. Why did you do that? Now we have two copies of identical scripts and have to worry about making fixes to both of them.

Comment (dmolokanov) [reply mqtt/build/windows/openssl.ps1]: My goal was to make a pipeline for broker up and running, without spending too much time polishing scripts that most likely will change soon. Now I see that scripts are way to different...

**Pull request merged**